Add sample club plan to hackclub/clubs

CoolBeans19 opened the pull request.

CoolBeans19 commented:
I reaaaally need some good feedback on this. I have a very basic start but I feel like we have to make this a lot better before merging.

zachlatta commented:
@aarohmankad, think you could take a look at this and leave some comments? This is meant as an example of a club plan we can give new leaders to treat as a sample.

I'm crazy busy right now and won't have a chance to review until tomorrow. Assigning @MaxWofford in the meantime while @aarohmankad is reviewing.

zachlatta commented:
@aarohmankad, let me know when you accept my invitation to join the Hack Club team on GitHub. I'll assign you to this issue then.

aarohmankad commented:
@zachlatta you can assign it to me now!

MaxWofford commented:
@aarohmankad Any way I can help out?

aarohmankad commented:
Overall it seems to follow a nice progressive structure. Maybe include a presentation after winter break of what they built?

Also I think you may have duplicated the Firebase workshop. If it isn't duplicated, i.e. takes three meetings to do, I wouldn't suggest working on that workshop with winter break in between. Kids will most likely forget what they learned, and recapping again would just waste time.

aarohmankad commented:
Also, from a usability standpoint, it would be nice if we made the items under the workshop column links to the actual projects.

aarohmankad commented:
thoughts @MaxWofford @zachlatta @CoolBeans19

MaxWofford commented:
+1 to not breaking up a workshop before and after break -- the chat workshop probably won't take 3 meetings anyways.

zachlatta commented on clubs/berkeley_high_plan.md:
```diff
@@ -0,0 +1,26 @@
+| 18    | Workshop                  | Get members familiar with Firebase and the concept of remote storage              | Firebase Chat     | 1. Recap guest speaker, introduce new workshop <br> 2. Let members go through workshop, offer help                                            |
+| 19    | Free form / presentation  | Have members implement remote storage into one of their projects                  | Firebase Chat     | 1. Brainstorm ft. sticky notes <br> 2. Look at ideas, form teams <br> 3. Work on projects <br> 4. Presentations / "I have failed"             |
+| 20    | Field Trip?               | Show club members a tech workspace and learn from professionals                   | None              | 1. Travel to the office by public transit <br> 2. Tour workspace <br> 3. Chat with developers <br> Leave for school                           |
+| 21    | FINALS WEEK - END OF SEMESTER |                                                                               |                   |                                                                                                                                               |
```
It might be easier to write this table as an HTML table.

zachlatta commented:
Looking good, deferring my review with @aarohmankad.

CoolBeans19 commented:
Okay, I have turned it from a markdown table to a HTML table as per @zachlatta's suggestion. Also added links to the workshop titles and removed erroneous duplicate of the Firebase workshop in favor of a longer, two-week final project. Want to do a final review and merge if you are satisfied @aarohmankad?

zachlatta commented on clubs/berkeley_high_plan.md:
```diff
@@ -0,0 +1,157 @@
+            <td>Workshop</td>
+            <td>Get members through the first workshop and teach them how easy it is to code!</td>
+            <td><a href="https://workshops.hackclub.com/personal_website/README.md">Personal Website</td>
+            <td>1. Recap Last meeting <br> 2. Guide members through first part of workshop <br> 3. Let members go through workshop, offer help</td>
```
FYI, you should be able to break this into multiple lines (instead of using `<br>`) now that you're using HTML tables.

CoolBeans19 replied:
Got it, lines are now different `<p>` tags

aarohmankad commented:
On your last real participation/free time week, I suggests giving them the link to all the hack club workshops, as there are a lot you don't go over and it's a resource for them to extend their learning.

zachlatta commented:
Any updates on this?

zachlatta commented:
I'm actually just going to step in here and merge this. @aarohmankad: can you submit a PR to add notes about those links on the last meeting?
